src/components/common/TmSessionExtension.vue

```diff
@@ -37,22 +37,33 @@
         </p>
       </div>
     </div>
+    <TmFormGroup class="field-checkbox" field-id="sign-up-warning" field-label>
+      <div class="field-checkbox-testnet">
+        <label class="field-checkbox-label" for="select-testnet">
+          <input id="select-testnet" v-model="testnet" type="checkbox" />
+          This is a testnet address</label
```

Comment:
@jbibla How about something like: Until we allow to copy addresses from testnet to mainnet you can decide the target network here.